Frends.Community.Pgp/Frends.Community.Pgp.csproj

```diff
@@ -11,7 +11,7 @@
     <IncludeSource>true</IncludeSource>
     <PackageTags>Frends</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Version>2.0.0</Version>
+    <Version>2.1.0</Version>
```

Comment:
This change only affects the task internally, so automatic update is possible, so I think you only have to increase build version. Please also update Changelog in README.md.